docs/db/changelog/changesets/powerauth-java-server/1.4.x/20230322-init-db.xml

```diff
@@ -310,6 +310,8 @@
     </changeSet>
 
     <changeSet id="17" logicalFilePath="powerauth-java-server/1.4.x/20230322-init-db.xml" author="Lubos Racansky">
+        <validCheckSum>9:3104c5b05d6dd090344e105faad0d13d</validCheckSum>
```

Comment:
This will not work with the currently supported liquibase version. But as we still do not incorporate this LB we can commit the checksums

Reply:
Ok, so let's remove the checksums.